contrib/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# docker build -t schedviz .
+# docker run -d -p 127.0.0.1:7402:7402 schedviz
+
+FROM debian
+
+USER root
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y build-essential unzip git
```

Comment:
Installing stuff prints errors about apt-utils not being installed.

Reply:
I'd ignored that message in past (its just a warning, based on the apt wanting to use an interactive debconfig frontend), but passing DEBIAN_FRONTEND=noninteractive via an ARG avoids it.

(Done).